src/lib/model/OutboundTransfersModel.js

```diff
@@ -215,15 +215,20 @@ class OutboundTransfersModel {
                     this.logger.push({ payee }).log('Payee resolved');
 
                     // stop listening for payee resolution messages
-                    this.cache.unsubscribe(payeeKey, subId).then(() => {
-                        // check we got the right payee and info we need
-                        if(payee.partyIdInfo.partyIdType !== this.data.to.idType) {
-                            const err = new Error(`Expecting resolved payee party IdType to be ${this.data.to.idType} but got ${payee.partyIdInfo.partyIdType}`);
-                            histTimerEnd({ success: false });
-                            resolvePayeeSpan.error(err.message);
-                            resolvePayeeSpan.finish(err.message);
-                            return reject(err);
-                        }
+                    // no need to await for the unsubscribe to complete.
+                    // we dont really care if the unsubscribe fails but we should log it regardless
+                    this.cache.unsubscribe(payeeKey, subId).catch(e => {
+                        this.logger.log(`Error unsubscribing (in callback) ${payeeKey} ${subId}: ${e.stack || util.inspect(e)}`);
+                    });
+
+                    // check we got the right payee and info we need
+                    if(payee.partyIdInfo.partyIdType !== this.data.to.idType) {
+                        const err = new Error(`Expecting resolved payee party IdType to be ${this.data.to.idType} but got ${payee.partyIdInfo.partyIdType}`);
+                        histTimerEnd({ success: false });
+                        esolvePayeeSpan.error(err.message);
```

Comment:
a typo or git visualisation not right?

Reply:
That is pretty weird it is correct on my side

Reply:
That was resolved, I had to redo the merge.